### PR TITLE
LPS-168242 [Raylife AP] Creating Active Claims table structure

### PR DESCRIPTION
**Pull request by EugenioAntunes**
Task: [LPS-168242](https://issues.liferay.com/browse/LPS-168242)

### CODE REVIEW

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import {ClayButtonWithIcon} from '@clayui/button';
+
+// import {ClayButtonWithIcon} from '@clayui/button';
+
+import ClayIcon from '@clayui/icon';
+import ClayTable from '@clayui/table';
+
+// import classnames from 'classnames';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:27]: Remove Comment

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+		key: 'fullName',
+		value: 'Name',
+	},
+	{
+		key: 'status',
+		value: 'Status',
+	},
+];
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:51]: Remove Comment

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+		date: '04/30/2022',
+		fullName: 'Maria Santos',
+		property: '2019 Ford Edge Se',
+		status: 'Pending Settlement',
+	},
+];
+
+const PolicyActiveClaims = () => {
+	return (
+		<>
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:78]: Remove Fragment here and at line 166

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+					<h5 className="m-0">Active Claims</h5>
+				</div>
+				<hr className="my-0" />
+				<ClayTable
+					borderedColumns={false}
+					borderless
+					className="tableeee w-100"
+					hover={false}
+				>
+					<ClayTable.Head>
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:90]: Add spaces

```suggestion
			<div className="bg-neutral policy-active-claims-container rounded">
				<div className="bg-neutral-0 policy-active-claims-title pt-3 px-5 rounded-top">
					<h5 className="m-0">Active Claims</h5>
				</div>
				
				<hr className="my-0" />
				
				<ClayTable
					borderedColumns={false}
					borderless
					className="tableeee w-100"
					hover={false}
				>
					<ClayTable.Head>
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+									key={index}
+								>
+									{header.value}
+								</ClayTable.Cell>
+							))}
+						</ClayTable.Row>
+					</ClayTable.Head>
+					<ClayTable.Body>
+						{activeClaimsData.map((rowContent, rowIndex) => (
+							<>
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:105]: Mapping an array without adding a `key` property to the first child will throw an error message in the console, like this: 

image

You can use Fragment Component to add a key

```suggestion
							<Fragment key={index}>
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+														<span className="p-1">
+															Next Step:
+														</span>
+													</strong>
+													<span className="m-0 p-1">
+														Review estimation for
+														816644
+													</span>
+												</div>
+
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:143]: Spaces

```suggestion
											<div className="d-flex justify-content-between p-1">
												<div className="align-items-center borderless col-5 d-flex pr-0">
													<span className="alert-indicator"></span>
													<strong className="lead m-0 p-1">
														<ClayIcon
															className="clay-icon-next p-0"
															symbol="info-circle"
														/>
														
														<span className="p-1">
															Next Step:
														</span>
													</strong>
													
													<span className="m-0 p-1">
														Review estimation for
														816644
													</span>
												</div>

```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+														className="m-0 p-1 view-detail-link"
+														href="#"
+													>
+														View Detail
+													</a>
+
+													<ClayButtonWithIcon
+														displayType={null}
+														symbol="times"
+													></ClayButtonWithIcon>
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:155]:
```suggestion
														className="m-0 p-1 view-detail-link"
														href="#"
													>
														View Detail
													</a>
													

													<ClayButtonWithIcon
														displayType={null}
														symbol="times"
													/>
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 8]
```diff
@@ -0,0 +1,169 @@
+/* eslint-disable @liferay/empty-line-between-elements */
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:1]: Remove Comment

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss [hunk 9]
```diff
@@ -0,0 +1,44 @@
+.policy-active-claims-container {
+	padding: 0px 200px;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss:2]: No need to add `px` to 0

```suggestion
	padding: 0 200px;
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+	@media screen and (max-width: 1500px) {
+		padding: 0px 50px;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss:5]:
```suggestion
		padding: 0 50px;
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+		background-position: 0 0;
+		background-repeat: repeat-x;
+		background-size: 6px 1px;
+		bottom: 0;
+		content: '';
+		height: 1px;
+		left: 0;
+		position: absolute;
+		right: 0;
+	}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss:43]: Spaces

```suggestion
	.policy-active-claims-title::after {
		background-color: var(--primary);
		content: '';
		display: block;
		height: 1px;
		margin-top: 5px;
		position: relative;
		width: 25px;
	}
	
	.policy-active-claims-title {
		padding-bottom: 10px;
	}
	
	a {
		color: #4bc286;
	}

	.header-border {
		border-bottom: 1px #aaa !important;
	}

	.info-row:not(:last-child) td::after {
		background-image: linear-gradient(
			to right,
			#aaa 0%,
			#aaa 20%,
			transparent 20%
		);
		background-position: 0 0;
		background-repeat: repeat-x;
		background-size: 6px 1px;
		bottom: 0;
		content: '';
		height: 1px;
		left: 0;
		position: absolute;
		right: 0;
	}
```

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss (hunk 10 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 8 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:1]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import './index.scss';
+
+import {ClayButtonWithIcon} from '@clayui/button';
+
+// import {ClayButtonWithIcon} from '@clayui/button';
```
[the remark this answers is not in the document]
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:20]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 1 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:27]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 2 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:51]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 3 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:78]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss (hunk 9 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss (hunk 10 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss (hunk 11 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss:43]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 7 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:155]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 4 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:90]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 2 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:51]: Done

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 13]
```diff
@@ -0,0 +1,153 @@
+/* eslint-disable @liferay/empty-line-between-elements */
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx:1]: Please remove that comment.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx [hunk 14]
```diff
@@ -0,0 +1,153 @@
...
+							</ClayTable.Row>
+							<ClayTable.Row className="info-row">
+								<ClayTable.Cell
+									className="border-0"
+									colSpan={5}
+								>
+									<div
+										className="alert alert-fluid alert-success w-100"
+										role="alert"
+									>
```
[the remark this answers is not in the document]
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx]: Done

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx (hunk 13 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.tsx]: Done

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Approved! Thank you @EugenioAntunes :white_check_mark:

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @EugenioAntunes

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**